FWCore/ParameterSet/python/Config.py

```diff
@@ -533,11 +533,17 @@ def extend(self,other,items=()):
                 if not item.hasLabel_() :
                     item.setLabel(name)
             elif isinstance(item,Schedule):
-                self.__setattr__(name,item)
+                if name == "schedule" and self.__schedule is not None:
+                    self.__schedule.extend(item)
+                else:
+                    self.__setattr__(name,item)
             elif isinstance(item,_Unlabelable):
                 self.add_(item)
             elif isinstance(item,ProcessModifier):
                 item.apply(self)
+            elif isinstance(item,Process):
```

Comment:
BTW, the reason for extending `extend()` and `load()` instead of adding a new `loadFragment()` method is to avoid having to change the mechanism used by cmsDriver & co to handle the cff files.
If you still prefer to have a different method, I will look into changing cmsDriver.